example/eblob_backend.c

```diff
@@ -872,13 +874,156 @@ int blob_defrag_start(void *priv, enum dnet_backend_defrag_level level)
 	return err;
 }
 
-int blob_defrag_stop(void *priv)
+static int blob_defrag_stop(void *priv)
 {
 	struct eblob_backend_config *c = priv;
 
 	return eblob_stop_defrag(c->eblob);
 }
 
+static int blob_send_reply(void *state, struct dnet_cmd *cmd, struct dnet_iterator_response *re, int more)
+{
+	int err;
+
+	dnet_convert_iterator_response(re);
+	err = dnet_send_reply_threshold(state, cmd, re, sizeof(struct dnet_iterator_response), more);
+
+	/* we have to convert response back, since it can be reused, for example like error response */
+	dnet_convert_iterator_response(re);
+	return err;
+}
+
+static int blob_send(struct eblob_backend_config *cfg, void *state, struct dnet_cmd *cmd, void *data)
+{
+	struct eblob_backend *b = cfg->eblob;
+	struct dnet_server_send_request *req = data;
+	struct dnet_raw_id *ids;
+	struct dnet_iterator_response re;
+	struct dnet_server_send_ctl *ctl;
+	int *groups;
+	int i, err;
+
+	struct dnet_ext_list elist;
+	static const size_t ehdr_size = sizeof(struct dnet_ext_list_hdr);
+	struct eblob_key key;
+	struct eblob_write_control wc;
+	uint64_t data_offset, record_offset;
+
+	dnet_ext_list_init(&elist);
+
+
+	/* structure has been already checked and has been proved to be correct no need to perform sanity checks */
+	dnet_convert_server_send_request(req);
+
+	ids = (struct dnet_raw_id *)(req + 1);
+	groups = (int *)(ids + req->id_num);
+
+	memset(&re, 0, sizeof(struct dnet_iterator_response));
+	re.total_keys = req->id_num;
+	re.flags = req->iflags;
```

Comment:
Why do you set `re.flags` to `req->iflags`? `re.flags` is designed to be set to iterated key flags from the backend.